util/unifyfs/src/unifyfs.c

```diff
@@ -76,6 +79,7 @@ static struct option const long_opts[] = {
     { "stage-in", required_argument, NULL, 'i' },
     { "stage-out", required_argument, NULL, 'o' },
     { "timeout", required_argument, NULL, 't' },
+    { "transfer-exe", required_argument, NULL, 'x' },
```

Comment:
@craigsteffen you forgot to add 'x:' to short_opts

Reply:
Indeed.  Thanks.  I'll roll that in with my next updated.